docs/UserGuide.md

```diff
@@ -92,40 +93,46 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the contact manager.
+Shows a list of all persons in the contact manager. Useful if you need to view all contacts that you have recorded.
 
 Format: `list`
 
+![list image](images/listImage.png)
+
 ### Editing a person : `edit`
 
-Edits an existing person in the contact manager.
+Edits an existing person in the contact manager. Useful if you need to edit a person's details to be updated, or if you have inputted a wrong field. Existing values will be updated to the input values.
 
 Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
+<box type="info" seamless>
+* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer**, such as 1, 2, 3, ...
+* At least one of the optional fields must be provided. This means that either `NAME`, `PHONE`, `EMAIL`, `ADDRESS` or `TAG` needs to be provided.
+* When editing tags, the existing tags of the person will be removed i.e., adding of tags is not cumulative.
+
+</box>
+
+<box type="tip" seamless>
 * You can remove all the person’s tags by typing `t/` without
     specifying any tags after it.
+</box>
 
 Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
 ### Locating persons by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons whose names contain any of the given keywords. Useful if you need to find a certain contact's details without going through the entire list of contacts.
 
 Format: `find [n/NAME] [a/ADDRESS] [t/TAG]`
 
 * `find` needs to have at least one of `n/NAME`, `a/ADDRESS`, or `t/TAG` to run.
-* The search is case-insensitive. e.g `hans` will match `Hans`
+* The search is case-insensitive. e.g., `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
 * Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Persons matching at least one keyword will be returned (i.e., it is an `OR` search).
+  * e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
```

Comment:
Include the case where multiple attributes are given; state that all given attributes must be matched for it to be returned.